backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/projection/util/SloppyMath.java

```diff
@@ -0,0 +1,295 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.search.projection.util;
```

Comment:
Please put this class in an `impl` package.

And maybe you could remove all the dead code? We only use part of this class, I think.

Reply:
Sure!